### PR TITLE
[minor][cdc-connector][mongodb] Add `op_type` metadata column

### PR DESCRIPTION
**Pull request by qg-lin**
Add `op_type` metadata column for mongodb source connector

### CODE REVIEW

**Comment by qg-lin** [comment]
@Jiabao-Sun @yuxiqian PTAL

File: flink-cdc-connect/flink-cdc-source-connectors/flink-connector-mongodb-cdc/src/test/java/org/apache/flink/cdc/connectors/mongodb/source/MongoDBFullChangelogITCase.java [hunk 1]
```diff
@@ -648,6 +648,7 @@ private void testMongoDBParallelSource(
                                 + " name STRING,"
                                 + " address STRING,"
                                 + " phone_number STRING,"
+                                + " op_type STRING METADATA FROM 'op_type' VIRTUAL,"
```
Comment (Jiabao-Sun) [inline flink-cdc-connect/flink-cdc-source-connectors/flink-connector-mongodb-cdc/src/test/java/org/apache/flink/cdc/connectors/mongodb/source/MongoDBFullChangelogITCase.java:651]: Thanks @qg-lin for this improvement.
How about add this case into `MongoDBConnectorITCase#testMetadataColumns`

File: docs/content.zh/docs/connectors/flink-sources/mongodb-cdc.md [hunk 2]
```diff
@@ -332,6 +332,11 @@ upstart 流需要一个唯一的密钥，所以我们必须声明 `_id` 作为
       <td>TIMESTAMP_LTZ(3) NOT NULL</td>
       <td>它指示在数据库中进行更改的时间。 <br>如果记录是从表的快照而不是改变流中读取的，该值将始终为0。</td>
     </tr>
+    <tr>
+      <td>op_type</td>
+      <td>STRING NOT NULL</td>
+      <td>该行的操作类型。</td>
```
Comment (yuxiqian) [inline docs/content.zh/docs/connectors/flink-sources/mongodb-cdc.md:338]: What about listing all possible values of `op_type` since it's enumerable?

**Comment by qg-lin** [comment]
Since DeduplicateFunctionHelper#processLastRowOnChangelog returns the row before deletion, the `delete` type is temporarily missing.

**Pull request closed without merging**